config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+      namespace :merchants do
+        get '/find', to: 'find#show'
```

Comment:
@icorson3 Is this kind of what you meant? Is it convention to use explicit routes for controllers that are not dealing with a model?

Reply:
Like it, changed mine to this too.